### DISCUSSION TITLE
What happens if I activate a conduit?

### DISCUSSION THREAD

**Discussion by anameeguy** [terrarium-earth/Ad-Astra#164] [General]
I haven't gotten the mod yet, but it looks interesting.
I was wondering if a conduit would give you air.
How would you supply water with oxygen?

**Comment by AlexNijjar** [comment]
it doesn't

**Reply by anameeguy** [reply]
Same with a nether portal?

**Reply by AlexNijjar** [reply]
nether portal doesn't work for modded dimensions I believe

**Comment by anameeguy** [comment]
That is interesting.
Does this mod work with create?

**Comment by AlexNijjar** [comment]
yes

**Reply by anameeguy** [reply]
Even with fluid pipes?

**Comment by AlexNijjar** [comment]
yes

**Reply by anameeguy** [reply]
Sweet.
What about origins?

**Comment by AlexNijjar** [comment]
like explicit support? no but it won't crash if that's what ur asking

**Reply by anameeguy** [reply]
Okay. Thanks.

**Reply by anameeguy** [reply]
Did you make this mod?
It says that you are a maintainer.

**Comment by AlexNijjar** [comment]
yes

**Reply by anameeguy** [reply]
Sweet.
I like to use fabric so I like it that this mod has support for it.
